lib/ash/resource/change/builtins.ex

```diff
@@ -131,6 +131,18 @@ defmodule Ash.Resource.Change.Builtins do
     @set_attribute_opts
   end
 
+  @doc """
+  Updates an existing attribute change by applying a function to it.
+
+  The update function gets called with the value already cast to the correct type, and only gets called
+  on valid changesets, so the value is guaranteed to have passed validations and constraints.
+  """
+  @spec update_change(attribute :: atom, update_function :: (any -> any)) ::
+          Ash.Resource.Change.ref()
+  def update_change(attribute, function) do
+    {Ash.Resource.Change.UpdateChange, attribute: attribute, function: function}
```

Comment:
Shouldn't `Spark.CodeHelpers.lift_functions` be used here like in other built-ins that accept functions?

Reply:
`set_argument` and `set_new_argument` accept functions but do not lift them, `after_action` etc do lift them.
I guess it would make sense to lift them here so the caller can define the function inline, which helps keep the relevant code in a single place

Reply:
Yeah, I think so 👍

Reply:
Done